recipes/rop/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: rop
+  version: 1.1
+
+source:
+  url: https://github.com/smangul1/rop/archive/v1.1.tar.gz
+  sha256: 5a76feb04a1e66c8e2f0fe1e70e03da9a3c7b87f7721bb0f2189e2fde3397224
+
+build:
+  number: 0
+  skip: True  # [not py27]
+
+requirements:
+  host:
+    - python=2.7
```

Comment:
You can remove the `=2.7` here. It will be added automatically via conda-build, and the skipping above already ensures that this only builds on python 2.7.

Reply:
Keeping the `=2.7` and removing `skip: true` would allow a `noarch: true` build.

Reply:
Removed the `=2.7`.